src/components/base/JLink/JLink.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+
+import React from 'react'
+import classnames from 'classnames'
+import { omit } from 'lodash-es'
+
+import jLinkStyle from './JLink.m.scss'
+import { type JLinkProps } from './types'
+import JLinkInternalWithRouter from './JLinkInternal'
+import {
+  isAnchor,
+  isExternal,
+} from './utils'
+
+export const JLink = (initialProps: JLinkProps) => {
+  const { href } = initialProps
+  const props = omit(initialProps, [
+    'theme',
+    'className',
+    'activeClassName',
+    'rel',
+  ])
+  const className = classnames(
+    initialProps.theme && jLinkStyle[initialProps.theme],
+    initialProps.className,
```

Comment:
would be great to destructure `theme` & `className` props as well as `href`

Reply:
Fixed